crates/orderbook/src/order_quoting.rs

```diff
@@ -407,14 +414,26 @@ impl OrderQuoter {
             fee_subsidy,
             storage,
             now: Arc::new(Utc::now),
+            eip1271_quote_validity_seconds,
+            presign_quote_validity_seconds,
```

Comment:
Maybe we can store these fields as `chrono::Duration` in the struct to not have to convert every time.